xbmc/cores/DllLoader/exports/util/EmuFileWrapper.cpp

```diff
@@ -264,7 +235,8 @@ bool CEmuFileWrapper::StreamIsEmulatedFile(FILE* stream)
 {
   if (isValidFilePtr(stream))
   {
-    return DescriptorIsEmulatedFile(FileDescriptor(*stream)->_file);
+    EmuFileObject* obj = reinterpret_cast<EmuFileObject*>(stream);
```

Comment:
The whole point of this PR is that it *never* is a real FILE ;)
If this assumption is proved false, obviously everything crumbles...